scripts/wheel/build_wheel.py

```diff
@@ -126,7 +126,10 @@ def generate_stubs(modules):
     os.chdir(WHEEL_ROOT_DIR)
     for module in modules:
         subprocess.check_call(
-            [*pybind11_stubgen_command, "--exit-code", "--output-dir", ".", f"meshlib.{module}"],
+            [*pybind11_stubgen_command,
+                # TODO uncomment this:
+                # "--exit-code",
```

Comment:
Has the time come?

Reply:
No, not yet.

Reply:
Report an issue about it too.

Reply:
https://github.com/MeshInspector/MeshInspectorCode/issues/4907